README.md

```diff
@@ -3,6 +3,47 @@
 
 This is where we make API calls from users viewing and making changes to the State Benefits section of their income tax return.
 
+## Running the service locally
+
+You will need to have the following:
+- Installed [MondoDB](https://docs.mongodb.com/manual/installation/)
+- Installed/configured [service manager](https://github.com/hmrc/service-manager).
+
+The service manager profile for this service is:
+
+    sm --start INCOME_TAX_STATE_BENEFITS
+
+Run the following command to start the remaining services locally:
+
+    sudo mongod (If not already running)
+    sm --start INCOME_TAX_SUBMISSION_ALL -r
+
+This service runs on port: `localhost:9377`
+
+### Downstream services
+All State Benefits data is retrieved / updated via the downstream system.
+- IF (Integration Framework)
```

Comment:
You should include income-tax-submission too

Reply:
Added